### PR TITLE
Fix typo in message

### PR DESCRIPTION
**Pull request by dg0yt**

### CODE REVIEW

File: locales/messages.en.json [hunk 1]
```diff
@@ -102,7 +102,7 @@
   "GraphCycleDetected": "Cycle detected within graph at {package_name}:",
```
Comment (ras0219-msft) [inline locales/messages.en.json:102]: Please avoid changes to the per-language JSONs and the lcl files -- those are all automatically regenerated by the translation team based on `messages.json`.

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM once the generated files changes are reverted. Thanks!

File: locales/messages.en.json (hunk 1 above)
Comment (dg0yt) [reply locales/messages.en.json]: Done.

**Comment by BillyONeal** [comment]
Thanks for the fix!

**Pull request merged**